{"name": "test_open_homepage_and_header", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be visible\nActual value: hidden \nCall log:\n  - LocatorAssertions.to_be_visible with timeout 5000ms\n  -   - waiting for locator(\"a.header__icon--account\").first\n  -     8 × locator resolved to <a data-no-instant=\"\" href=\"https://theconnectedshop.com/customer_authentication/redirect?locale=en&region_country=UA\" class=\"header__icon header__icon--account link header__button-icon flex justify-center align-center\">…</a>\n  -       - unexpected value \"hidden\"", "trace": "self = <test_homepage.TestHeaderAndSearch object at 0x00000282C90ABCE0>\nopen_connected_shop = <Page url='https://theconnectedshop.com/'>\n\n    @allure.story(\"Open homepage and validate header\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_open_homepage_and_header(self, open_connected_shop):\n        page = open_connected_shop\n        header = Header(page)\n    \n        with allure.step(\"Check homepage URL and title\"):\n            assert page.url.startswith(\"https://theconnectedshop.com\")\n            assert \"The Connected Shop\" in page.title()\n    \n        with allure.step(\"Validate logo in header\"):\n            header.validate_logo()\n    \n        with allure.step(\"Validate account link in header\"):\n>           header.validate_account_link()\n\ntests\\test_homepage.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.header.Header object at 0x00000282C94A2960>\n\n    @allure.step(\"Validate account link in header\")\n    def validate_account_link(self):\n>       expect(self.account_link).to_be_visible(timeout=5000)\nE       AssertionError: Locator expected to be visible\nE       Actual value: hidden \nE       Call log:\nE         - LocatorAssertions.to_be_visible with timeout 5000ms\nE         -   - waiting for locator(\"a.header__icon--account\").first\nE         -     8 × locator resolved to <a data-no-instant=\"\" href=\"https://theconnectedshop.com/customer_authentication/redirect?locale=en&region_country=UA\" class=\"header__icon header__icon--account link header__button-icon flex justify-center align-center\">…</a>\nE         -       - unexpected value \"hidden\"\n\npages\\header.py:17: AssertionError"}, "steps": [{"name": "Check homepage URL and title", "status": "passed", "start": 1755893192928, "stop": 1755893193127}, {"name": "Validate logo in header", "status": "passed", "steps": [{"name": "Validate logo in header", "status": "passed", "start": 1755893193127, "stop": 1755893193187}], "start": 1755893193127, "stop": 1755893193187}, {"name": "Validate account link in header", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be visible\nActual value: hidden \nCall log:\n  - LocatorAssertions.to_be_visible with timeout 5000ms\n  -   - waiting for locator(\"a.header__icon--account\").first\n  -     8 × locator resolved to <a data-no-instant=\"\" href=\"https://theconnectedshop.com/customer_authentication/redirect?locale=en&region_country=UA\" class=\"header__icon header__icon--account link header__button-icon flex justify-center align-center\">…</a>\n  -       - unexpected value \"hidden\"\n\n", "trace": "  File \"C:\\Users\\BestLaptop\\Downloads\\Project\\Connectedtheshop\\tests\\test_homepage.py\", line 25, in test_open_homepage_and_header\n    header.validate_account_link()\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\BestLaptop\\Downloads\\Project\\Connectedtheshop\\pages\\header.py\", line 17, in validate_account_link\n    expect(self.account_link).to_be_visible(timeout=5000)\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 20199, in to_be_visible\n    self._sync(self._impl_obj.to_be_visible(visible=visible, timeout=timeout))\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 677, in to_be_visible\n    await self._expect_impl(\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "steps": [{"name": "Validate account link in header", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be visible\nActual value: hidden \nCall log:\n  - LocatorAssertions.to_be_visible with timeout 5000ms\n  -   - waiting for locator(\"a.header__icon--account\").first\n  -     8 × locator resolved to <a data-no-instant=\"\" href=\"https://theconnectedshop.com/customer_authentication/redirect?locale=en&region_country=UA\" class=\"header__icon header__icon--account link header__button-icon flex justify-center align-center\">…</a>\n  -       - unexpected value \"hidden\"\n\n", "trace": "  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\BestLaptop\\Downloads\\Project\\Connectedtheshop\\pages\\header.py\", line 17, in validate_account_link\n    expect(self.account_link).to_be_visible(timeout=5000)\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 20199, in to_be_visible\n    self._sync(self._impl_obj.to_be_visible(visible=visible, timeout=timeout))\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 677, in to_be_visible\n    await self._expect_impl(\n  File \"C:\\Users\\BestLaptop\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "start": 1755893193188, "stop": 1755893198215}], "start": 1755893193188, "stop": 1755893198216}], "start": 1755893192928, "stop": 1755893198218, "uuid": "6263fc35-d918-4918-b2fd-5add0619470f", "historyId": "e11f48281c1b2d482f6e17f9cec63d54", "testCaseId": "e11f48281c1b2d482f6e17f9cec63d54", "fullName": "tests.test_homepage.TestHeaderAndSearch#test_open_homepage_and_header", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "Homepage"}, {"name": "story", "value": "Open homepage and validate header"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_homepage"}, {"name": "subSuite", "value": "TestHeaderAndSearch"}, {"name": "host", "value": "DESKTOP-TPM9FFM"}, {"name": "thread", "value": "59568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_homepage"}], "titlePath": ["tests", "test_homepage.py", "TestHeaderAndSearch"]}